src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -186,29 +198,75 @@ export class ActivityFeedItem extends React.Component<Props> {
     return (smartWalletStatus.status !== SMART_WALLET_UPGRADE_STATUSES.DEPLOYMENT_COMPLETE);
   }
 
+  findAccount = (address: string) => {
```

Comment:
We have the `findAccountById` util for that

Reply:
And it could be worth adding the `addressesEqual` comparison into the `findAccountById()`

Reply:
Or you could use `findAccountByAddress()`